frontend/src/models/user.model.ts

```diff
@@ -0,0 +1,31 @@
+export enum IRole {
+    Candidate,
+    Participant,
+    Mentor,
+    Admin,
+};
+
+export enum IStatus {
```

Comment:
What does mean 'I' letter in names IStatus and IRole?
In interfaces 'I' suggests that this is name of interface, but in enum types I dont know what it means.

Reply:
You're right, it shouldn't be like that, it introduces confusion and means absolutely nothing - I changed it. Thanks ;)